### PR TITLE
Feature monitor

### PR DESCRIPTION
**Pull request by Threated**
This is a first draft of the task progress monitoring at proxy level

### CODE REVIEW

**Review by TKussel — CHANGES_REQUESTED** [review]
Please add some lines of documentation to the Readme

File: proxy/monitor/README.md
[the remark this answers is not in the document]
Comment (Threated) [reply proxy/monitor/README.md:1]: Yeah I forgot to remove the boilerplate Readme that the inital codegen generates

**Comment by Threated** [comment]
I will write something about this in the Readme once #123 is merged as only than I will know how the final CI and docker image name will look like. I will than need to rebase this branch as there were a lot of changes since this code was written.

**Comment by Threated** [comment]
As there have been some developments regarding adding more app specific visualization to the monitoring I suggest to extract the frontend of this PR into its own application and just provide the required endpoint.

**Comment by TKussel** [comment]
I fully agree. A separation between a) the publication of the event stream (this PR) and b) the visualization of the data (in a separate repository) is a smart idea to keep Beam's responsibilities concise.

**Comment by Threated** [comment]
Closed in favor of [beam-inspector](https://github.com/samply/beam-inspector) and [this branch](https://github.com/samply/beam/tree/feature/beam-inspector)

**Pull request closed without merging**